### ISSUE TITLE
Starting the game with the injector crashes

### ISSUE THREAD

**Issue by jcotton42** [InsanityBringer/InjectD3#8]
```
---------------------------
Error
---------------------------
Excecution in WinMain() was stopped by an Illegal Instruction


Call stack:
------------------------------------------------------------------
    main.exe 0056fa7c()
    KERNEL32.DLL 766a6739()
    ntdll.dll 775190af()
    ntdll.dll 7751907d()
------------------------------------------------------------------

[ This info has been copied to the clipboard for pasting to a bug report. ]


Registers:
EAX=00000001 CS=0023 EIP=0055fdb0 EFLGS=00010216
EBX=0019ca44 SS=002b ESP=0019c4a4 EBP=00000000
ECX=0019c48c DS=002b ESI=ffffffff FS=0053
EDX=00000027 ES=002b EDI=00000000 GS=002b

---------------------------
OK   
---------------------------
```

Settings I'm using
![image](https://user-images.githubusercontent.com/5257450/185718614-81411ce2-d661-4c34-a747-95a36cf2d820.png)
![image](https://user-images.githubusercontent.com/5257450/185718617-a94dbb1e-48cb-481d-a18d-014a3d0c91e2.png)

I'm using the Steam version of Descent 3, SHA256 hash of Descent 3.exe: CEF1E372523D9D2196F9127976ACC90BBF7014EFE8A86E95680B034582C4BA6E

I tried completely reinstalling the game, but the issue persists.

I'm on Windows 11 x64 21H2, version 22000.856

My GPU is an RTX 3080, driver version 516.94.

Here's a DxDiag export https://pastebin.com/UWv2CKCv

**Comment by jcotton42** [comment]
I was told on the /r/descent Discord to change the UI framerate back to 20, and that fixed it.

Is this feature known to be broken, or did I just get unlucky?

**Comment by InsanityBringer** [comment]
Late response, but I tracked down the bug, and it is fixed in commit 9d80b46
